test/gtest/Makefile.am

```diff
@@ -68,11 +69,10 @@ gtest_CPPFLAGS = \
 gtest_LDFLAGS  = $(GTEST_LDFLAGS) -no-install -Wl,-dynamic-list-data
 gtest_CFLAGS   = $(BASE_CFLAGS)
 gtest_CXXFLAGS = \
-	$(BASE_CXXFLAGS) $(GTEST_CXXFLAGS) \
+	$(BASE_CXXFLAGS) $(GTEST_CXXFLAGS) -std=c++11 \
```

Comment:
you mentioned cpp11 is not supported by gcc4, so how does it work?
does it mean only some of cpp11 features are supported?

Reply:
C++11 **is** supported by gcc 4 (and indeed the pipeline succeeds).
We can't use a newer version of googletest, because it requires gcc >= 5 (mentioned in their docs).
Gtest 1.11 can't compile with gcc 4, one of their requirements is `std::is_trivially_copy_constructible`.

Reply:
ok